drivers/input/ajoystick.c

```diff
@@ -835,7 +748,6 @@ int ajoy_register(FAR const char *devname,
   return OK;
 
 errout_with_priv:
```

Comment:
I think we can remove `errout_with_priv` here and just `kmm_free(priv);` in `if (ret < 0)` instead.

Reply:
Done.